code/identity/src/main/java/com/adobe/marketing/mobile/identity/IdentityExtension.java

```diff
@@ -666,6 +666,12 @@ void loadVariablesFromPersistentData() {
         }
         mid = namedCollection.getString(IdentityConstants.DataStoreKeys.MARKETING_CLOUD_ID, null);
 
+        Log.trace(
+                IdentityConstants.LOG_TAG,
+                LOG_SOURCE,
+                "Load ECID from persistence: ecid = %s",
+                mid);
```

Comment:
can we consolidate all the logs from this method in a single line, including the ecid value?